src/inc/quic_datapath.h

```diff
@@ -434,8 +434,7 @@ typedef enum CXPLAT_DATAPATH_FEATURES {
     CXPLAT_DATAPATH_FEATURE_TCP                = 0x00000020,
     CXPLAT_DATAPATH_FEATURE_RAW                = 0x00000040,
     CXPLAT_DATAPATH_FEATURE_TTL                = 0x00000080,
-    CXPLAT_DATAPATH_FEATURE_SEND_DSCP          = 0x00000100,
-    CXPLAT_DATAPATH_FEATURE_RECV_DSCP          = 0x00000200,
+    CXPLAT_DATAPATH_FEATURE_SEND_DSCP          = 0x00000100
```

Comment:
keep the trailing comma so that the next addition doesn't have to touch this line.

Reply:
done